### DISCUSSION TITLE
Ajout du support d'EcoleDirecte

### DISCUSSION THREAD

**Discussion by Baraban511** [PapillonApp/papillon-v5#107] [Q&A] [answered]
### Description du problème

Je ne l’ai pas vu sur la roadmap, et je ne sais pas si c’est possible / facile à faire mais le support d’Ecole Directe va t’il être ajouté à Papillon IOS ?

### Description de l'amélioration

Support EcoleDirecte

**Comment by moeenio** [comment, marked as answer]
La prise en charge de EcoleDirecte est en cours de développement, le travail est mené par @LeGeek01 et @LeMaitre4523. Le développement s'effectue sur [cette branche de ce fork](https://github.com/LeMaitre4523/Papillon/tree/service/ecoledirecte) et une partie a déja été intégrée dans la branche service/ecoledirecte du dépôt officiel. On peut suivre leur avancement et s'impliquer dans le salon #ecoledirecte sur le Discord de la communauté.

**Comment by LeGeek01** [comment]
Je confirme

**Comment by moeenio** [comment]
Il confirme

**Comment by LeGeek01** [comment]
je confirme que je confirme

**Comment by Baraban511** [comment]
Merci pour vos confirmation !
